### PR TITLE
Navbar Component

### PR DESCRIPTION
**Pull request by Abhiram716**
### Creating Navbar Component

### CODE REVIEW

**Review by aadilmughal786 — APPROVED** [review]

File: src/components/Header/HeaderItem.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import { Box, ButtonBase, Typography, useTheme } from "@mui/material";
```
Comment (aadilmughal786) [inline src/components/Header/HeaderItem.jsx:1]: We should separate local and global import for better readability

global comes first then local imports

File: src/components/Header/HeaderItem.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import React from "react";
+
+const HeaderItem = ({
+  activeItem, //state variable
```
Comment (aadilmughal786) [inline src/components/Header/HeaderItem.jsx:5]: Seems like we can remove comments

File: src/components/Header/index.jsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  
+  const handleItemClick = (item) => {
+    setActiveItem(item);
+  };
+
+  const matchXs = useMediaQuery((theme) => theme.breakpoints.down("md"));
+  const theme = useTheme();
+
+  return (
+    <>
```
Comment (aadilmughal786) [inline src/components/Header/index.jsx:24]: Seems like we can remove this fragments

File: src/components/Header/index.jsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+              variant="h4"
+              sx={{ ml: 14, mb: 3, mt: 3 }}
+              color={theme.palette.primary.main}
+            >
+              Abhiram Kantipudi
+            </Typography>
+          )}
+          <Toolbar sx={{ mb: 3, mt: 3 }}>
+            <HeaderItem
+              activeItem={activeItem}
```
Comment (aadilmughal786) [inline src/components/Header/index.jsx:45]: Code seems like duplicate here we can make a list of item and then iteratively render them we need to keep our code DRY as much as possible

File: src/pages/index.jsx [hunk 5]
```diff
@@ -3,11 +3,13 @@ import Container from "@mui/material/Container";
 import CssBaseline from "@mui/material/CssBaseline";
 import React from "react";
 import theme from "../theme/theme";
```
Comment (aadilmughal786) [inline src/pages/index.jsx:5]: We should separate local and global import for better readability

also in other places

File: src/theme/theme.js [hunk 6]
```diff
@@ -1,9 +1,18 @@
 import { createTheme } from "@mui/material/styles";
```
Comment (aadilmughal786) [inline src/theme/theme.js:1]: We should separate local and global import for better readability

File: src/components/Header/HeaderItem.jsx (hunk 1 above)
Comment (Abhiram716) [reply src/components/Header/HeaderItem.jsx:1]: Can you explain a bit more about this?
Do you mean I create a separate file for Importing components related to MUI?

**Review by aadilmughal786 — APPROVED** [review]

**Pull request merged**